.gitignore

```diff
@@ -5,6 +5,7 @@
 /tmp
 /out-tsc
 /test
+/.swc
```

Comment:
Since lingui extractor uses swc, it produces ./.swc/plugins/<uuid>-<arch> etc file which is cache and we dont need to version control it